google/youtube/tool.gpt

```diff
@@ -6,35 +6,24 @@ Share Tools: Transcribe YouTube Video
 
 ---
 Name: Transcribe YouTube Video
-Description: Get a text transcript of a YouTube video and clean it up.
-Param: video_url: The URL of a YouTube video. (e.g. https://www.youtube.com/watch?v=VIDEO_ID or https://youtu.be/VIDEO_ID)
-Tool: Fetch Raw YouTube Transcript, Generate Raw YouTube Transcript, github.com/gptscript-ai/context/workspace
-Share Tools: github.com/gptscript-ai/context/workspace
-
-First, take the video_url parameter and pass it to the Fetch Raw YouTube Transcript tool to get a raw transcript.
-If that tool fails, then try calling the Generate Raw YouTube Transcript tool to generate a raw transcript.
-Process that transcript and add the missing punctuation. Clean up any misspellings or errors in the transcript.
-Do not truncate or reword any of the text, just add punctuation and correct mistakes.
-Write the corrected text to a file in the $GPTSCRIPT_WORKSPACE_DIR with a filename based on the VIDEO_ID.
-Let the user know where the file is located, and give a short 2 sentence summary of the video.
-
----
-Name: Fetch Raw YouTube Transcript 
-Metadata: index: false
 Description: Get YouTube's raw text transcript of a video.
-Param: video_url: The URL of a YouTube video. (e.g. https://www.youtube.com/watch?v=VIDEO_ID or https://youtu.be/VIDEO_ID)
+Share Context: YouTube Context
+Param: video_url: The URL of a YouTube video. (e.g. URLs in the form https://www.youtube.com/watch?v=<VIDEO_ID> or https://youtu.be/<VIDEO_ID>). Does not accept live stream URLs (e.g. URLs starting vith https://www.youtube.com/live/).
 
 #!/usr/bin/env python3 ${GPTSCRIPT_TOOL_DIR}/transcriber.py
 
 ---
-Name: Generate Raw YouTube Transcript
```

Comment:
If we are deleting this tool outright, we should also delete the code.

Reply:
Done!